### PR TITLE
Lint warnings #5

### PR DESCRIPTION
**Pull request by a-palchikov**
## Description

This PR focuses on lint changes in package lib/utils.

## Type of change

* Internal change (not necessarily a bug fix or a new feature)

## TODOs

- [x] Self-review the change
- [ ] Perform manual testing
- [ ] Address review feedback

### CODE REVIEW

**Review by bernardjkim — APPROVED** [review]

File: lib/utils/helm/helmutils.go [hunk 1]
```diff
@@ -41,7 +41,7 @@ func Vals(valueFiles valueFiles, values []string, stringValues []string, fileVal
 	return yaml.Marshal(base)
 }
 
-func merge(valueFiles valueFiles, values []string, stringValues []string, fileValues []string, CertFile, KeyFile, CAFile string) (map[string]interface{}, error) {
+func merge(valueFiles valueFiles, values []string, stringValues []string, fileValues []string, _, _, _ string) (map[string]interface{}, error) {
```
Comment (bernardjkim) [inline lib/utils/helm/helmutils.go:44]: Can we just remove the last three parameters?

Comment (a-palchikov) [reply lib/utils/helm/helmutils.go:44]: Yes.

**Comment by a-palchikov** [comment]
@wadells There seems to be an [issue](https://drone.teleport.dev/gravitational/gravity/68/1/9) with disk space on CI:
```
[ERROR]: write /drone/src/e/build/cache/gravity-state/7.0.13/packages/tmp/blob648694514: no space left on device
```

**Comment by wadells** [comment]
> @wadells There seems to be an [issue](https://drone.teleport.dev/gravitational/gravity/68/1/9) with disk space on CI:
> 
> ```
> [ERROR]: write /drone/src/e/build/cache/gravity-state/7.0.13/packages/tmp/blob648694514: no space left on device
> ```

Yeah, I noticed that yesterday, and started working on it.  The issue is we don't specify/Drone doesn't support ephemeral-storage requirements for its job pods.  I'm looking into if we can switch volume types or simply throw bigger mounts at it in the short term.

**Pull request merged**